model/schema/core.yaml

```diff
@@ -355,71 +356,74 @@ classes:
       - prefix_order
 
   Chromosome:
-    is_a: AuditedObject
-    description: The ID of the landmark used to establish the coordinate system for the current feature.
+    is_a: GenomicEntity
```

Comment:
Should this really be a GenomicEntity?  If we want something to represent the idealized biological entity across different assemblies then will it have a curie, for example?

Reply:
Using the 1920s definition, I would say yes, but from a modern practical definition, perhaps it's more confusing so I'll just change it to BiologicalEntity for now.

Reply:
BiologicalEntity objects still require a curie field though - will these have a curie?

Reply:
Sounds good, and some sort of name/label slot